app/models/service_template_ansible_tower.rb

```diff
@@ -27,4 +43,48 @@ def self.default_reconfiguration_entry_point
   def self.default_retirement_entry_point
     nil
   end
+
+  def self.validate_config_info(options)
+    config_info = options[:config_info]
+    unless config_info[:configuration_script_id] || config_info[:configuration]
+      raise _('Must provide configuration_script_id or configuration')
+    end
+    config_info
+  end
+  private_class_method :validate_config_info
+
+  def create_resource_actions(ae_endpoints)
```

Comment:
@jntullo This method does not appear to differ from the one in the parent ServiceTemplate model here: [app/models/service_template.rb#create_resource_actions](https://github.com/ManageIQ/manageiq/blob/a9343f486d68982b392d048c67adc5e5b16cc42e/app/models/service_template.rb#L309)

Is this really needed?